ui/src/consts/difficultyLevel.ts

```diff
@@ -0,0 +1,7 @@
+import i18next from '../i18n';
+
+export const difficultyLevel = {
+  1: i18next.t('tasks.difficultyLevels.easy'),
+  2: i18next.t('tasks.difficultyLevels.medium'),
```

Comment:
ogólnie jeśli chcemy używać tutaj translacji to lepiej byłoby używać tego jako funkcji. Ten obiekt stworzy się tylko raz, jeśli zmienimy zmienilibyśmy język w aplikacji funkcja t nie zostanie wywołana ponownie i powstanie nam tutaj bug. Jeśli chcemy trzymać to tutaj to możemy zrobić coś na zasadzie 

```
export const difficultyLevel = () => {
    id: 1,
    label: i18next.t('tasks.difficultyLevels.easy'),
  }
```

Lub żeby pozostawić to po prostu jako obiekt to 
```
export const difficultyLevel = () => {
    id: 1,
    labelKey: 'tasks.difficultyLevels.easy',
  }
```

i wtedy podczas używania tego obiektu w komponencie używać ten key w wywołaniu funkcji t